src/jobs/time_entries_sync_job.ts

```diff
@@ -287,10 +288,27 @@ export class TimeEntriesSyncJob extends SyncJob {
         // loop through all STEOs (except that which TE is updated last)
         for (const serviceTimeEntryObjectWrapper of timeEntrySyncedObjectWrapper.serviceTimeEntryObjectWrappers) {
           if (serviceTimeEntryObjectWrapper !== lastUpdatedServiceTimeEntryObjectWrapper) {
-            // delete current TE from the service and continue to the scenario e)
-            await serviceTimeEntryObjectWrapper.syncedService.deleteTimeEntry(serviceTimeEntryObjectWrapper.serviceTimeEntryObject.id);
-            // set as undefined => use scenario e)
-            serviceTimeEntryObjectWrapper.timeEntry = undefined;
+
+            if (serviceTimeEntryObjectWrapper.timeEntry) {
+              console.log("B1 - update")
+
+              const index = timeEntrySyncedObjectWrapper.timeEntrySyncedObject.serviceTimeEntryObjects.indexOf(serviceTimeEntryObjectWrapper.serviceTimeEntryObject);
+              timeEntrySyncedObjectWrapper.timeEntrySyncedObject.serviceTimeEntryObjects.splice(index, 1);
```

Comment:
nechceme sem pridat nieco ako if (index === -1) check?
Predpokladam, ze by nemalo nastat, ale radsej to mat pod kontrolou.

Reply:
lognut na sentry info